atmos.yaml

```
@@ -351,3 +351,9 @@ settings:
   #            If the source and destination lists have the same length, all items in the destination lists are
   #            deep-merged with all items in the source list.
   list_merge_strategy: replace
+
+vendor:
+  # Path to the vendor.yaml file
+  # Supports both absolute and relative paths
+  # Can also be set using 'ATMOS_VENDOR_YAML_PATH' ENV var, or '--vendor-yaml-path' command-line argument
+  vendor_yaml_path: "path/to/your/vendor.yaml"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)